auth/user.go

```diff
@@ -652,7 +652,7 @@ func (user *userImpl) FilterToAvailableCollectionChannels(scope, collection stri
 		if channelName == ch.AllChannelWildcard {
 			return user.InheritedCollectionChannels(scope, collection).Copy(), nil
 		}
-		added := filtered.AddChannel(channelName, user.canSeeCollectionChannelSince(scope, collection, channelName))
+		added := filtered.AddChannel(channelName, user.canSeeCollectionChannelSince(scope, collection, channelName), "")
```

Comment:
should this be using the value of Source of the added channel from `canSeeCollectionChannelSince`?